rockchip-replace.xml

```diff
@@ -1,46 +1,46 @@
-  <project groups="pdk" name="android_bootable_recovery" path="bootable/recovery" remote="rk-aosp"/>
-  <project groups="pdk-cw-fs,pdk-fs" name="android_frameworks_base" path="frameworks/base" remote="rk-aosp"/>
+  <project path="system/memory/libion" name="android_system_memory_libion" groups="pdk" remote="rk-aosp"/>
+  <project path="bootable/recovery" name="android_bootable_recovery" groups="pdk" remote="rk-aosp"/>
+  <project path="frameworks/base" name="android_frameworks_base" groups="pdk-cw-fs,pdk-fs" remote="rk-aosp"/>
   <project path="hardware/broadcom/libbt" name="android_hardware_broadcom_libbt" groups="pdk" remote="rk-aosp"/>
-  <project groups="pdk,broadcom_wlan" name="android_hardware_broadcom_wlan" path="hardware/broadcom/wlan" remote="rk-aosp" />
-  <project groups="pdk" name="android_external_wpa_supplicant_8" path="external/wpa_supplicant_8" remote="rk-aosp" />
+  <project path="hardware/broadcom/wlan" name="android_hardware_broadcom_wlan" groups="pdk,broadcom_wlan" remote="rk-aosp" />
+  <project path="external/wpa_supplicant_8" name="android_external_wpa_supplicant_8" groups="pdk" remote="rk-aosp" />
   <project path="frameworks/av" name="android_frameworks_av" groups="pdk"  remote="rk-aosp"/>
-  <project groups="pdk" name="android_frameworks_opt_net_wifi" path="frameworks/opt/net/wifi" remote="rk-aosp" />
+  <project path="frameworks/opt/net/wifi" name="android_frameworks_opt_net_wifi" groups="pdk" remote="rk-aosp" />
   <project path="frameworks/native" name="android_frameworks_native" groups="pdk" remote="rk-aosp"/>
   <project path="external/skia" name="android_external_skia" groups="pdk,qcom_msm8x26" remote="rk-aosp"/>
   <project path="external/tinyalsa" name="android_external_tinyalsa" groups="pdk" remote="rk-aosp"/>
   <project path="kernel/configs" name="android_kernel_configs" remote="rk-aosp" >
-    <linkfile dest="kernel/kernel/configs/android-11.config" src="rockchip/android-11.config"/>
-    <linkfile dest="kernel/kernel/configs/android-11-go.config" src="rockchip/android-11-go.config"/>
-    <linkfile dest="kernel/kernel/configs/rockchip3326.config" src="rockchip/rockchip3326.config"/>
-    <linkfile dest="kernel/kernel/configs/rockchip356x.config" src="rockchip/rockchip3326.config"/>
-    <linkfile dest="kernel/kernel/configs/rockchip356x_evb.config" src="rockchip/rockchip356x_evb.config"/>
-    <linkfile dest="kernel/kernel/configs/rockchip356x_eink.config" src="rockchip/rockchip356x_eink.config"/>
-    <linkfile dest="kernel/kernel/configs/rockchip3399.config" src="rockchip/rockchip3399.config"/>
-    <linkfile dest="kernel/kernel/configs/non_debuggable.config" src="r/android-4.19/non_debuggable.config"/>
+    <linkfile src="rockchip/android-11.config" dest="kernel/kernel/configs/android-11.config" />
+    <linkfile src="rockchip/android-11-go.config" dest="kernel/kernel/configs/android-11-go.config" />
+    <linkfile src="rockchip/rk3326.config" dest="kernel/kernel/configs/rk3326.config" />
```

Comment:
This is where builds were failing. It was trying to link the non-existant rockchip3326, etc files to the configs directory.